api/turing/models/autoscaling_policy.go

```diff
@@ -21,11 +22,13 @@ const (
 )
 
 type AutoscalingPolicy struct {
-	Metric AutoscalingMetric `json:"metric" validate:"required,oneof=concurrency rps cpu memory"`
+	Metric *AutoscalingMetric `json:"metric"`
```

Comment:
With this change, we are not testing the oneOf, right? We shouldn't remove everything and I think `omitempty` instead of `required` can help:

`json:"metric" validate:"omitempty,oneof=concurrency rps cpu memory"`

This could help trim down some of the special validation logic in `api/turing/validation/validator.go`